src/Raven.Server/Monitoring/Snmp/Objects/Server/1.6/ServerTotalMemory.cs

```diff
@@ -1,19 +1,33 @@
-using System.Diagnostics;
+using System;
 using Lextm.SharpSnmpLib;
+using Sparrow;
+using Sparrow.LowMemory;
+using Sparrow.Platform;
+using Sparrow.Platform.Posix;
 
 namespace Raven.Server.Monitoring.Snmp.Objects.Server
 {
     public class ServerTotalMemory : ScalarObjectBase<Gauge32>
     {
+        private readonly Lazy<SmapsReader> _smapsReader;
```

Comment:
Is this thread safe?

Reply:
Very unlikely to have an issue here but... https://github.com/ravendb/ravendb/pull/9663